openmc/mgxs/mgxs.py

```diff
@@ -5011,13 +5028,1153 @@ class NuFissionMatrixXS(MatrixMGXS):
     def __init__(self, domain=None, domain_type=None,
                  groups=None, by_nuclide=False, name='', num_polar=1,
                  num_azimuthal=1):
-        super(NuFissionMatrixXS, self).__init__(domain, domain_type,
-                                                groups, by_nuclide, name,
-                                                num_polar, num_azimuthal)
-        self._rxn_type = 'nu-fission'
-        self._hdf5_key = 'nu-fission matrix'
+        super(ScatterProbabilityMatrix, self).__init__(
+            domain, domain_type, groups, by_nuclide,
+            name, num_polar, num_azimuthal)
+
+        self._rxn_type = 'scatter'
+        self._estimator = 'analog'
+        self._valid_estimators = ['analog']
+        self._hdf5_key = 'scatter probability matrix'
+
+    @property
+    def scores(self):
+        scores = ['scatter']
+        return scores
+
+    @property
+    def filters(self):
+        # Create the non-domain specific Filters for the Tallies
+        group_edges = self.energy_groups.group_edges
+        energy = openmc.EnergyFilter(group_edges)
+        energyout = openmc.EnergyoutFilter(group_edges)
+        filters = [[energy, energyout]]
+        return self._add_angle_filters(filters)
+
+    @property
+    def rxn_rate_tally(self):
+        if self._rxn_rate_tally is None:
+            self._rxn_rate_tally = self.tallies['scatter']
+            self._rxn_rate_tally.sparse = self.sparse
+        return self._rxn_rate_tally
+
+    @property
+    def xs_tally(self):
+
+        if self._xs_tally is None:
+            energyout_bins = \
+                [self.energy_groups.get_group_bounds(i) for i in range(self.num_groups, 0, -1)]
```

Comment:
This would look better as:
```Python
energyout_bins = [self.energy_groups.get_group_bounds(i)
                  for i in range(self.num_groups, 0, -1)]
```